builtin/providers/aws/resource_aws_api_gateway_domain_name.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsApiGatewayDomainName() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate_upload_date": &schema.Schema{
```

Comment:
Well, it is already done looking at `https://github.com/Ninir/terraform/blob/6655e0f628e9d9b877aefe438a8ee63c17c45dfa/builtin/providers/aws/resource_aws_api_gateway_domain_name.go#L108` 

thank you again :)